tasks/gulp/watch.js

```diff
@@ -8,5 +8,5 @@ const configPaths = require('../../config/paths.json')
 gulp.task('watch', () => {
   gulp.watch([configPaths.src + '**/**/*.scss', configPaths.app + 'assets/scss/**/*.scss'], ['styles', 'sassdoc'])
   gulp.watch([configPaths.src + '**/**/*.js'], ['scripts'])
-  // gulp.watch([configPaths.src + '**/**/*.njk'], ['generate:readme', 'compile:components'])
+  // gulp.watch([configPaths.src + '**/**/*.njk'], ['compile:components'])
```

Comment:
Given this is commented out, should we just remove this line? (non-blocking)

Reply:
Sure will do